### ISSUE TITLE
checkpoints.corrupted

### ISSUE THREAD

**Issue by thiner** [mrhan1993/Fooocus-API#42]
**Describe the problem**
The `models/checkpoints` folder is renamed to `models/checkpoints.corrupted` while API start to work. But the checkpoint files are all good in fact. Because it works in Fooocus by removing the suffix `corrupted` from the folder name.  I think this is a bug of model files validation.

**Comment by EvilBT** [comment]
把default_refiner_model_name = ''替换成default_refiner_model_name = 'None'，不要'', 要'None'
    if name == 'None':
        print(f'Refiner unloaded.')
        return

**Comment by konieshadow** [comment]
> **Describe the problem** The `models/checkpoints` folder is renamed to `models/checkpoints.corrupted` while API start to work. But the checkpoint files are all good in fact. Because it works in Fooocus by removing the suffix `corrupted` from the folder name. I think this is a bug of model files validation.

Are you using realistic preset? Or manually set `default_refiner_model_name` to empty string?

**Comment by thiner** [comment]
> > **Describe the problem** The `models/checkpoints` folder is renamed to `models/checkpoints.corrupted` while API start to work. But the checkpoint files are all good in fact. Because it works in Fooocus by removing the suffix `corrupted` from the folder name. I think this is a bug of model files validation.
> 
> Are you using realistic preset? Or manually set `default_refiner_model_name` to empty string?

No, I didn't use the realistic preset.
I copied `user_path_config.txt` from Fooocus, and the relevant settings as below:
```
"default_model": "juggernautXL_version6Rundiffusion.safetensors",
"default_refiner": "None",
```
Fooocus is running well with these settings.

**Comment by thiner** [comment]
By the way, my Fooocus version:
`Fooocus version: 2.1.774`
Could it be a version conflict?

**Comment by thiner** [comment]
> 把default_refiner_model_name = ''替换成default_refiner_model_name = 'None'，不要'', 要'None' if name == 'None': print(f'Refiner unloaded.') return

Thanks for your answer, but I didn't modify any code. I just copied `user_path_config.txt` from Fooocus.

**Comment by konieshadow** [comment]
> By the way, my Fooocus version: `Fooocus version: 2.1.774` Could it be a version conflict?

It should not. It seems like an issue with Fooocus-API, and it will be fix it soon.

**Comment by konieshadow** [comment]
@thiner Fixed by Fooocus-API [v0.3.7.](https://github.com/konieshadow/Fooocus-API/releases/tag/v0.3.7)

**Comment by thiner** [comment]
Verified and it's fixed.

Thanks!
